api/src/main/java/com/github/eyefloaters/console/api/errors/client/ConstraintViolationHandler.java

```diff
@@ -23,7 +23,7 @@
 @Provider
 public class ConstraintViolationHandler implements ExceptionMapper<ConstraintViolationException> {
 
-    Logger logger = Logger.getLogger("com.github.eyefloaters.console.api.errors.client");
+    Logger logger = Logger.getLogger(ConstraintViolationHandler.class);
```

Comment:
If you're using a constant expression for the logger name you might want to make the `logger` `static final`?

Reply:
Changed constant class-based loggers to `static final`.